include/functions.h

```diff
@@ -1148,7 +1148,7 @@ s32 Camera_GetInputDirYaw(Camera* camera);
 s16 Camera_GetCamDirPitch(Camera* camera);
 s16 Camera_GetCamDirYaw(Camera* camera);
 void func_800DFD04(Camera* camera, s32 arg1, s32 arg2, s32 arg3);
-// void func_800DFD78(void);
+void func_800DFD78(Camera* camera, UNK_TYPE arg1, Actor* actor);
```

Comment:
```suggestion
void func_800DFD78(Camera* camera, s32 paramFlag, void* value);
```
This is the correct prototype for this. In OoT/MM, this function is currently called `Camera_SetParam`, which sets a group of generic data. You can use this to set the target actor like you do in `z_player_lib`, but you can also set Vec3f `eye`/`at` data or `f32` roll/fov data as an example:

https://github.com/engineer124/mm/blob/1e8b04396788f52b6e942cb3e9f1def396c4484a/src/code/z_camera_part3.c#L993-L996